x-pack/test/functional/apps/machine_learning/feature_controls/ml_security.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from 'expect.js';
```

Comment:
Need to merge master and rewrite this import before merging: https://github.com/elastic/kibana/pull/33761

Reply:
This PR is to merge it into the `granular-app-privileges` branch, we're gonna have to update a ton of these when doing so...